tests/quickcheck.rs

```diff
@@ -0,0 +1,34 @@
+#[macro_export]
+macro_rules! quickcheck {
+    (
+        $(
+            $(#[$m:meta])*
+            fn $fn_name:ident($($arg_name:ident : $arg_ty:ty),*) -> $ret:ty {
+                $($code:tt)*
+            }
+        )*
+    ) => {
+        $(
+            #[test]
+            $(#[$m])*
+            fn $fn_name() {
+                fn prop($($arg_name: $arg_ty),*) -> $ret {
+                    $($code)*
+                }
+
+                // Use QUICKCHECK_TESTS from compiletime to surpass miri isolation
+                let tests = option_env!("QUICKCHECK_TESTS").and_then(|s| s.parse().ok());
+                #[cfg(miri)]
+                let tests = tests.or(Some(1));
+
+                let mut qc = ::quickcheck::QuickCheck::new();
+
+                if let Some(tests) = tests {
+                    qc = qc.tests(tests)
+                }
```

Comment:
Rather than defaulting to 1 if `QUICKCHECK_TESTS` is not supplied, can we default to whatever `quickcheck`'s default is, and set `QUICKCHECK_TESTS=1` in the CI env for MIRI?

Reply:
Well, yes. That's what this code does.